HW3_spark_smythe.py

```diff
@@ -1,156 +0,0 @@
-"""
-  Created by Ed Smythe Sept 2020
-  BDA 696 @ SDSU
-  Current run time is ~ xxxx
-  Baseball ⚾
-  Load this dataset into Spark directly from MariaDB
-  Calculate the batting average using an Transformer
-  Rolling (over last 100 days)
-  Look at the last 100 days that player was in prior to this game
-  Hints
-  You already did it earlier, use that logic to your advantage
-  Make sure the calculation works first
-  Then worry about wrapping it up as a transformer
-  Only get the needed raw data from MariaDB
-  No calculations in MariaDB
-  Do all the logic in Spark
-"""
-
-import sys
-
-from pyspark import StorageLevel, keyword_only
-from pyspark.ml import Pipeline, Transformer
-from pyspark.ml.classification import RandomForestClassifier
-from pyspark.ml.feature import CountVectorizer
-from pyspark.ml.param.shared import HasInputCols, HasOutputCol
-from pyspark.ml.util import DefaultParamsReadable, DefaultParamsWritable
-from pyspark.sql import SparkSession
-from pyspark.sql.functions import col, concat, lit, split, when
-from split_column_transform import SplitColumnTransform

```

Comment:
You don't need 'from split_column_transform import SplitColumnTransform' unless you have the class 'SplitColumnTransform' as another file. Please see the comment below.

Reply:
ah ok